db/schema.sql

```diff
@@ -71,10 +71,12 @@ CREATE TABLE aggregation_jobs(
     aggregation_job_id BYTEA NOT NULL,                  -- 32-byte AggregationJobID as defined by the PPM specification
     aggregation_param  BYTEA NOT NULL,                  -- encoded aggregation parameter (opaque VDAF message)
     state              AGGREGATION_JOB_STATE NOT NULL,  -- current state of the aggregation job
+    lease_expiry       TIMESTAMP,                       -- when lease on this aggregation job expires; NULL if no current lease
```

Comment:
Sure; I have to reset the lease expiry to `-infinity` on release rather than `NULL` which is a little funky, but things work out OK enough. I might revise this later if I create a "lease token" that includes a randomness -- at that point I think it makes sense to record (NULL, NULL) when there is no lease -- but for now this is definitely a little cleaner.